application/src/test/java/org/opentripplanner/street/model/edge/ElevatorHopEdgeTest.java

```diff
@@ -87,4 +100,27 @@ private State[] traverse(Accessibility wheelchair, StreetSearchRequest req) {
 
     return edge.traverse(state);
   }
+
+  @Test
+  void testDuration() {
+    var way = new OsmWay();
+    way.addTag("duration", "00:01:02");
+    way.addTag("highway", "elevator");
+    var provider = TestOsmProvider.of().addWay(way).build();
+    var graph = new Graph(new Deduplicator());
+    var osmModule = OsmModule.of(
+      provider,
+      graph,
+      new DefaultOsmInfoGraphBuildRepository(),
+      new DefaultVehicleParkingRepository()
+    ).build();
+    osmModule.buildGraph();
+    var edges = graph.getEdgesOfType(ElevatorHopEdge.class);
+    assertThat(edges).hasSize(2);
+    var edge = (ElevatorHopEdge) edges.getFirst();
+    var from = edge.getFromVertex();
+    var req = StreetSearchRequest.of().withMode(StreetMode.WALK);
+    var res = edge.traverse(new State(from, req.build()))[0];
+    assertEquals(62_000, res.getTimeDeltaMilliseconds());
```

Comment:
This PR is fine as it is but where do you stand about allowing getters just for tests instead of testing the traversal logic in the same test?

If you are fine with it, this can be split in two tests, but it's not a requirement for my approval.